login/login.go

```diff
@@ -218,7 +223,7 @@ func (login *DeviceLogin) Login() error {
 	var err error
 	login.wellKnown, err = login.getWellKnown()
 	if err != nil {
-		return fmt.Errorf("failed to fetch .well-known configuration: %v", err)
+		return err
```

Comment:
I have preference for keeping.

```suggestion
		return fmt.Errorf("failed to fetch .well-known configuration: %v", err)
```